contrib/README_joao_pessoa.md

```
@@ -0,0 +1,90 @@
+# João Pessoa (PB) Scraper
+
+Scrapers for extracting parish Mass and confession schedules from Arquidiocese da Paraíba.
+
+**Website:** https://arquidiocesepb.org.br/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)